src/gatling/simulations/uk/gov/hmcts/ccd/corecasedata/simulations/CCD_PerformanceRegression.scala

```diff
@@ -287,24 +288,35 @@ class CCD_PerformanceRegression extends Simulation  {
   }
 
 	setUp(
-		// API_ProbateCreateCase.inject(simulationProfile(testType, probateTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-		// API_CMCCreateCase.inject(simulationProfile(testType, cmcTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-		// API_DivorceCreateCase.inject(simulationProfile(testType, divorceTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-		// API_IACCreateCase.inject(simulationProfile(testType, iacTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),		
-    // CaseActivityListScn.inject(rampUsers(500) during (10 minutes)),		
-		// CaseActivityScn.inject(rampUsers(500) during (10 minutes)),
-    // CCDSearchView.inject(rampUsers(200) during (20 minutes)),		
-		// CCDElasticSearch.inject(rampUsers(300) during (20 minutes)), //300 during 20
-
-    API_CMCCaseEvents.inject(simulationProfile(testType, cmcTargetPerHour, numberOfPipelineUsers)).pauses(pauseOption),
-
-
-		// CaseActivityListScn.it(simulationProfile(testType, searchTargetPerHour/searchRepeatsPerUser, numberOfPipelineUsers)).pauses(pauseOption),  	
+     //simulation for cdm-test-performance repo
```

Comment:
Some of the indentation in the simulation file seems a little funny

Reply:
@jonsoloway-hmcts it had been a select copy and comment.  fixed to be inline in the follow up commit.